.circleci/config.yml

```diff
@@ -64,13 +62,21 @@ workflows:
   version: 2
   build_and_test:
     jobs:
-      - build
-      - version-bump
-      # Uncomment the lines below when you have a Pypi account and you are ready to deploy your country package to reusers.
-      # - deploy:
-      #     requires:
-      #       - build
-      #     filters:
-      #       # owner: ServiceInnovationLab  # prevent failed builds on forks # This filter is not valid yet in CircleCI 2.0, see <https://discuss.circleci.com/t/20443>
-      #       branches:
-      #         only: master
+      - build:
+          filters: #Note: The build job must also have a filters tags section, as it is a transient dependency of the deploy job.
+            tags:
+              only: /.*/
+      - version-bump:
+          requires:
+            - build
+          filters: #Note: The version-bump must also have a filters tags section, as it is a transient dependency of the deploy job.
+            tags:
+              only: /.*/
+      - deploy:
+          requires:
+            - build
+          filters:
+            tags:
+              only: /.*/
+            branches:
+              ignore: /.*/
```

Comment:
I understand that as “publish only tags, ignore all branches”, which I interpret as “never deploy” 🤔 I feel it should be `branches: only: master`.